kobo/hub/models.py

```diff
@@ -123,6 +123,8 @@ class Meta:
     def __unicode__(self):
         return u"%s" % self.name
 
+    def __str__(self):
+        return u"%s" % self.name
```

Comment:
It should return `str` on python2 and not unicode:

```suggestion
        return "%s" % self.name
```

(Well that's the minimal change, but "%s" % (x) seems to be a bit of an antipattern anyway, shouldn't it be str(x) ?  Why use string formatting if the format is just "%s" ?)